frontend/packages/console-shared/src/utils/utils.ts

```diff
@@ -1,7 +1,9 @@
+import { createContext } from 'react';
 import { JSONSchema7 } from 'json-schema';
 import { toPath } from 'lodash';
+import * as _ from 'lodash';
```

Comment:
You can merge this two lodash imports and import just what you need:

```suggestion
import { startCase, toPath } from 'lodash';
```